fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/TypeRdfContext.java

```diff
@@ -58,71 +61,36 @@ public TypeRdfContext(final FedoraResource resource,
 
         //include rdf:type for primaryType, mixins, and their supertypes
         concatRdfTypes();
+        if (resource() instanceof NonRdfSource) {
+            // gather versionability info from the parent
+            if (resource().getNode().getParent().isNodeType("mix:versionable")) {
```

Comment:
I am suggesting that the versionable property has in the past been stored on the description because it was the description that held the properties... which were being versioned.
Now, the properties are stored on the NonRdfSource, so it would therefore seem to make sense that we no longer care if the description is versioned, but rather if the NonRdfSource is versioned.